### DISCUSSION TITLE
WMS Support

### DISCUSSION THREAD

**Discussion by anilkrky** [DotSpatial/DotSpatial#1536] [Q&A]
Hello. I want to use dotspatial for working both shapefiles and WMS. But I am not sure if it is supporting WMS (some discussions states that it should be updated). Is there any examples for WMS or WEBMAP plugin?

**Comment by sindizzy** [comment]
There is an examples folder in the source. Have you checked there?

**Reply by anilkrky** [reply]
Yes, I checked. There isn't any example for Web controls. Probably I need to check the library. I hope everything works :)
